src/ZoomLoop.Api/Features/Profiles/ProfileExtensions.cs

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) Quinntyne Brown. All Rights Reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using ZoomLoop.Core.Models;
+
+namespace ZoomLoop.Api.Features.Profiles;
+
+public static class ProfileExtensions
+{
+    public static ProfileDto ToDto(this Profile profile)
+    {
+        return new ProfileDto
+        {
+            ProfileId = profile.ProfileId,
+            ProfileImageUrl = profile.ProfileImageUrl,
+            FirstName = profile.FirstName,
+            LastName = profile.LastName,
+            PhoneNumber = profile.PhoneNumber,
+            DateOfBirth = profile.DateOfBirth,
+            HomeAddress = profile.HomeAddress.ToDto()
+        };
+    }
+
+    public static AddressDto ToDto(this Address address)
```

Comment:
Create a class called AddressExtensions and move this extension method to that class